docs/user-guide.md

```diff
@@ -2069,6 +2069,14 @@ You can apply Oracle Release Update (RU) or Patch Set Update (PSU) patches to
 both the Grid Infrastructure and Database homes by using the
 `apply-patch.sh` script of the toolkit.
 
+By default, `install-oracle.sh` updates to the latest available patch.  To
+apply a specific patch instead, use the `--no-patch` option in `intall-oracle.sh`
```

Comment:
Typo: in**s**tall-oracle.sh

Otherwise, lgtm.

Reply:
Updated;  ptal.